### ISSUE TITLE
Repeat Rows Feature

### ISSUE THREAD

**Issue by MorlaRamakrishna** [nicolaskruchten/pivottable#853]
Can we add repeat row labels feature to Pivot JS same like as MS Excel Pivot. Reference please look in to below url
[Repeat Row Labels](https://www.youtube.com/watch?v=N4vw5INrIjE)

**Comment by nicolaskruchten** [comment]
I don't think this is a feature I would like to add to the main library, but you can copy and modify the `TableRenderer` to do this without too much effort I would guess.
